python/py-terminaltables/Portfile

```diff
@@ -24,7 +27,13 @@ checksums           rmd160  9d02523f89d32c890ae9c40212155e221584e279 \
                     size    12264
 
 if {${name} ne ${subport}} {
-    depends_build-append    port:py${python.version}-setuptools
-
-    livecheck.type  none
+    # this patch is from git-format-patch,
+    # which requires flag `-p1`, instead of
+    # the macports portpatch default of `-p0`.
+    patch.pre_args  -p1
+    patchfiles-append \
+                    0001-switch-to-poetry-core.patch
+
+    depends_build-append \
+                    port:py${python.version}-poetry-core
```

Comment:
this dependency is added automatically by the `python` PG when you set the `pyhon.pep517_backend` to `poetry` ; so please remove

Reply:
Done